tools/updateShinyCSS.R

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env Rscript
+library(rprojroot)
+library(sass)
+
+scss <- find_package_root_file("inst/www/shared/shiny_scss/shiny.scss")
+shiny_css <- sass(
+  sass_file(scss),
+  options = sass_options(output_style = "expanded", source_comments = FALSE)
+)
+writeLines(shiny_css, "inst/www/shared/shiny.css")
```

Comment:
The dest path should also use rprojroot.

We should have a unit test to make sure the files match, so that we don't ship a wrong version of the css file. Need to watch out for `\n` issues on Windows though. Might be easiest to use `brio::write_lines(eol = "\n")`

The tools/README.md file should also have a mention of this script.

Reply:
Seems like we could/should workaround that by doing the check via `git status` (like we do for built JS files), so I did that in 1c99f3d

Reply:
The reason for the separate JS check in GHA is that it requires outside resources (namely node and node packages). I think it makes sense to add this as a unit test since it only involves R code, and uses packages that are already installed. (bootstraplib has similar tests for the precompiled assets.)

Looking at the GHA node_js workflow, running this script there makes it so it needs to install/compile a bunch of R packages on every run in order to do the test when it's part of the node_js workflow. We might as well just do the test as part of the unit tests because the cost there is low -- instead of installing a bunch of packages, it just needs to compile the scss file and compare it to the css file.

One other thought: having the test in the unit tests will also ensure that someone the exact same output will be generated on all platforms (so that someone developing on Windows won't commit a file that's different from what would be generated on Linux/Mac).